Wordle.Api/Wordle.Api/Identity/Policies.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using System.Security.Claims;
+using System.ComponentModel;
+
+namespace Wordle.Api.Identity;
+public static class Policies
+{
+	public const string RandomAdmin = "RandomAdmin";
+	public const string MasterOfTheUniverse = "MasterOfTheUniverse";
+	public const string OlderThanTwentyOne = "OlderThanTwentyOne";
+
+	public static void RandomAdminPolicy(AuthorizationPolicyBuilder policy)
+	{
+		policy.RequireRole(Roles.Admin);
+		policy.RequireAssertion(context =>
+		{
+			var random = context.User.Claims.FirstOrDefault(c => c.Type == Claims.Random);
+			if (Double.TryParse(random?.Value, out double result))
+			{
+				return result > 0.5;
+			}
+			return false;
+		});
+	}
+
+	public static void MasterOfTheUniversePolicy(AuthorizationPolicyBuilder policy)
+	{
+		policy.RequireAssertion(context =>
+		{
+			var motuClaim = context.User.Claims.FirstOrDefault(c => c.Type == Claims.MasterOfTheUniverse);
+			if (Boolean.TryParse(motuClaim?.Value, out bool motu))
+			{
+				return motu;
+			}
+			return false;
+		});
+	}
```

Comment:
```suggestion
	public static void MasterOfTheUniversePolicy(AuthorizationPolicyBuilder policy)
	{
policy.RequireClaim(Claims.MasterOfTheUniverse, "true");
	}
```

Since the value of `MasterOfTheUniverse` will either be true or false, we don't need to do a lot of processing of it. Having it be an actual bool I think won't matter since it going to either the string value of 'true` or `false`.

Reply:
That's really smart. I didn't know RequireClaim was a method. Thank you for showing me that!